src/js/pages/UniversePage.js

```diff
@@ -1,7 +1,21 @@
+import {MountService} from 'foundation-ui';
 import React from 'react';
 
 import Icon from '../components/Icon';
 
+const IntroductoryText = () => {
```

Comment:
I very much like that you used the mount module as this already prepares the extraction of a universe package, but wonder if we should move this into a separate file.